contracts/TranscodeJobs.sol

```diff
@@ -63,6 +66,8 @@ library TranscodeJobs {
         self.jobs[self.numJobs].transcoderAddress = _electedTranscoder;
         self.numJobs++;
 
+        NewJob(_electedTranscoder, msg.sender);
```

Comment:
We can't listen for library events, but as a workaround if the library emits an event that is also defined in the contract that uses the library, you can listen for the event via the contract and you will be notified when the library emits the event